e2e/vmss.go

```diff
@@ -231,7 +231,7 @@ func getBaseVMSSModel(name, sshPublicKey, customData, cseCmd string, opts *scena
 	return armcompute.VirtualMachineScaleSet{
 		Location: to.Ptr(config.Location),
 		SKU: &armcompute.SKU{
-			Name:     to.Ptr("standard_d2s_v4"),
+			Name:     to.Ptr("Standard_DS2_v2"),
```

Comment:
do we not need to change the line `defaultAgentPoolVMSize     = "standard_d2s_v4"` in cluster.go?

Reply:
that dictates the vm size for the system agentpool  from which we run the jumpbox pods - those are only created when new clusters are provisioned, thus I don't think we need to

Reply:
if it's still an issue we can always change in another follow-up

Reply:
I think it would be cleaner and lest confusion to centralized the SKU to that constant.